types/roads/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "export-just-namespace": false
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`

Reply:
ok @plantain-00, could you tell me please how can I do this without `"export-just-namespace": false` .
because this doen't work which is that suggest tslint
`export namespace roads`

I don't want something like this `import * as roads from "roads"`
Just `import {middleware, Road, Respond} from "roads"`